src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/package.json

```diff
@@ -46,6 +47,69 @@
     ],
     "main": "./dist/extension",
     "contributes": {
+        "breakpoints": [
+          {
+            "language": "aspnetcorerazor"
+          },
+          {
+            "language": "csharp"
+          }
+        ],
+        "debuggers": [
+          {
+            "type": "blazorwasm",
+            "label": "Blazor WebAssembly Debug",
+            "initialConfigurations": [
+              {
+                "type": "blazorwasm",
+                "name": "Launch and Debug Blazor WebAssembly Application",
+                "request": "launch"
+              }
+            ],
+            "configurationAttributes": {
+              "launch": {
+                "properties": {
+                  "cwd": {
+                    "type": "string",
+                    "description": "The directory of the Blazor WASM app, defaults to the workspace folder.",
+                    "default": "${workspaceFolder}"
+                  },
+                  "env": {
+                    "type": "object",
+                    "description": "Environment variables passed to dotnet"
+                  },
+                  "url": {
+                    "type": "string",
+                    "description": "The URL of the application",
+                    "default": "https://localhost:5001"
+                  },
+                  "browser": {
+                    "type": "string",
+                    "description": "The debugging browser to launch (Edge or Chrome)",
+                    "default": "chrome",
+                    "enum": ["chrome", "edge"]
+                  },
+                  "trace": {
+                    "type": ["boolean", "string"],
+                    "default": "true",
+                    "enum": ["verbose", true],
+                    "description": "If true, verbose logs sent to log file. If 'verbose', send logs to console."
```

Comment:
This is an underlying config supported by the JS debuggers so I've reflected its requirements.

I believe verbose defaulting to send to a log file is done to avoid excessive spew in the console since it is quite noisy.